poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
```

Comment:
Sven - thanks for the update! That's exactly what I meant to clarify. :)

On Thu, Jun 4, 2015 at 5:42 PM, Sven Marnach <notifications@github.com>
wrote:

> In poll/public/handlebars/poll_results.handlebars
> <https://github.com/mckinseyacademy/xblock-poll/pull/13#discussion_r31768930>
> :
>
> > @@ -0,0 +1,43 @@
> > +<script id="poll-results-template" type="text/html">
> > +    <h3 class="poll-header">{{display_name}}</h3>
> > +    <div class="poll-question-container">{{{question}}}</div>
> > +    <ul class="poll-answers-results poll-results">
> > +    {{#each tally}}
> > +        <li class="poll-result">
> > +            <div class="poll-result-input-container">
> > +              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
>
> @sarina <https://github.com/sarina> Sorry, the comment "Done" is
> primarily addressed to myself, to keep track of what has already been
> addressed and what's still open. There are still quite a few points open.
> When I'm done with everything, I'll update the sandbox and let you know.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mckinseyacademy/xblock-poll/pull/13/files#r31768930>.
>